module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,17 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> list) {
+    std::deque<std::string> deque;
+    std::copy(list.begin(), list.end(), std::back_inserter(deque));
```

Comment:
You can create a deque in that way:
```suggestion
    std::deque<std::string> words ( list.begin(), list.end() );
```